Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,47 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
+    system "./configure"
+    system "make"
+    # install all executable files in "main" directory
+    Dir.foreach("main") do |entry|
+      path = "main/#{entry}"
+      if File.file?(path) && File.executable?(path)
+        bin.install path
+      end
+    end
+  end
+
+  test do
+    # define some variables
+    rate = 16000
+    freq = 100
+    dur = 5
+    bn = 'sine'
+    txtfile = "#{bn}.txt"
+    wavfile = "#{bn}.wav"
+    ptcfile = "#{bn}.ptc"
+
+    # create 5 s, 16 kHz sampled, 100 Hz sine wave as ASCII file
+    File.open(txtfile, 'w') do |f|
+      scale = 2 ** 15 - 1
+      f.puts Array.new(dur * rate) { |i| (scale * Math.sin(freq * 2 * Math::PI * i / rate)).to_i }
+    end
+
+    # convert to wav format using ch_wave
+    %x(ch_wave #{txtfile} -itype raw -istype ascii -f #{rate} -o #{wavfile} -otype riff)
+
+    # pitch tracking to est format using pda
+    %x(pda #{wavfile} -shift #{1 / freq.to_f} -o #{ptcfile} -otype est)
```

Comment:
Yeh, you can use backticks to capture output (we prefer them to `%x`) but system for those above.